src/test/ui/unsized/expect_unsized_return_sized.rs

```diff
@@ -0,0 +1,8 @@
+trait Foo<'x> {}
```

Comment:
Nit: If you add `// compile-flags: --crate-type lib`, you can remove the "`main` function not found" error.

Reply:
Done